### PR TITLE
refactoring #42 - global structural refactoring

### PR DESCRIPTION
**Pull request by winniepukki**
Fixes: winniepukki/trinat#42

### CODE REVIEW

File: src/public/manifest.json [hunk 1]
```diff
@@ -1,17 +1,42 @@
...
   "orientation": "portrait-primary",
   "icons": [
     {
-      "src": "",
-      "type": "",
-      "sizes": ""
+      "src": "./assets/icons/icon-72x72.png",
+      "type": "image/png",
+      "sizes": "72x72"
+    },
```
Comment (elinapeshelee) [inline src/public/manifest.json:15]: Please provide the code with the `apple-touch-icon`

File: webpack.config.js [hunk 2]
```diff
@@ -65,6 +67,7 @@ module.exports = {
         new CopyWebpackPlugin({
             patterns: [
                 { from: './src/public/assets/images/', to: 'assets/img' },
+                { from: './src/public/assets/icons/', to: 'assets/icons' },
                 { from: './src/public/assets/favicon', to: 'assets/favicon' },
                 { from: './src/public/manifest.json', to: './[name].[ext]' }
             ]
```
Comment (elinapeshelee) [inline webpack.config.js]: This should be destructured from the `paths` variables from above in the future

File: webpack.config.js [hunk 3]
```diff
@@ -65,6 +67,7 @@ module.exports = {
         new CopyWebpackPlugin({
```
Comment (elinapeshelee) [inline webpack.config.js:67]: The `alias: {}` object lacks the `@type` directory. This should be provided

**Review by elinapeshelee — CHANGES_REQUESTED** [review]
Please pay attention to the improvement tips ♥️

File: webpack.config.js (hunk 3 above)
Comment (winniepukki) [reply webpack.config.js:67]: Good tips. Thank you! 😊

File: webpack.config.js (hunk 2 above)
Comment (winniepukki) [reply webpack.config.js]: This is a great tip that will be taken into the consideration in the upcoming PR!!

**Review by elinapeshelee — APPROVED** [review]
Looks legit

**Pull request merged**